tests/test_relation.py

```diff
@@ -0,0 +1,339 @@
+import pytest
+from inspect import getmembers
+import re
+import pandas
+import numpy as np
+import datajoint as dj
+from datajoint.table import Table
+from unittest.mock import patch
+
+from . import schema
+
+
+@pytest.fixture
```

Comment:
Since these fixtures are used in many different modules, we'll move them to `conftest.py` after migrating everything.